### ISSUE TITLE
Are Scala 2.13 versions published anywhere?

### ISSUE THREAD

**Issue by worace** [krasserm/streamz#87]
Apologies if I am missing something obvious, but I was not able to find Scala 2.13 artifacts in bintray or maven central. Are these actually published, or is it still just Scala 2.12 for the moment? The Bintray directory I was investigating is here FWIW: https://dl.bintray.com/krasserm/maven/com/github/krasserm/.

Thanks!

**Comment by Daenyth** [comment]
At minimum, 0.13 is published for 2.12

Check the readme, the bintray group changed

**Comment by worace** [comment]
Ah, thanks @Daenyth I had absolutely missed that.

For anyone else who makes my same mistake, had to change the custom bintray resolver:

```diff
-    resolvers += Resolver.bintrayRepo("krasserm", "maven"),
+    resolvers += Resolver.bintrayRepo("streamz", "maven"),
```
